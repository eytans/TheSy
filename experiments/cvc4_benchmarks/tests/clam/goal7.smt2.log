19:28:16 [WARN] Collecting statistics
19:28:16 [WARN]  ex is: zero
19:28:16 [WARN]  ex is: (succ zero)
19:28:16 [WARN]  ex is: (succ (succ zero))
19:28:16 [WARN]  ex is: nil
19:28:16 [WARN]  ex is: (cons autovar_Nat_0 nil)
19:28:16 [WARN]  ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:28:16 [WARN] Running exploration without proof mode, but goals were given
19:28:16 [WARN] Proof mode: true
19:28:16 [WARN]  ex is: zero
19:28:16 [WARN]  ex is: (succ zero)
19:28:16 [WARN]  ex is: (succ (succ zero))
19:28:16 [WARN]  ex is: nil
19:28:16 [WARN]  ex is: (cons autovar_Nat_0 nil)
19:28:16 [WARN]  ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:28:16 [WARN] Running exploration without proof mode, but goals were given
19:28:16 [WARN] Proof mode: true
19:28:16 [WARN] proved: (plus ?ts_ph_Nat_0 zero) => ?ts_ph_Nat_0
19:28:16 [WARN] proved: (leq zero ?ts_ph_Nat_0) => true
19:28:16 [WARN] proved: (append ?ts_ph_Lst_0 nil) => ?ts_ph_Lst_0
19:28:39 [WARN] proved: (plus ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1)) => (succ (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1))
19:28:39 [WARN] proved: (succ (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1)) => (plus ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1))
19:28:39 [WARN] Found all lemmas
