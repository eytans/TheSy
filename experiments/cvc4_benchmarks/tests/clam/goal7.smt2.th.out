19:28:16 [0m[33m[WARN] [0mCollecting statistics
(leq ?__x0 ?__y1) => (or (= ?__x0 ?__y1) (less ?__x0 ?__y1))
(or (= ?__x0 ?__y1) (less ?__x0 ?__y1)) => (leq ?__x0 ?__y1)
(plus zero ?n) => ?n
(plus (succ ?n) ?m) => (succ (plus ?n ?m))
(succ (plus ?n ?m)) => (plus (succ ?n) ?m)
(append nil ?x) => ?x
(append (cons ?x2 ?y) ?z) => (cons ?x2 (append ?y ?z))
(cons ?x2 (append ?y ?z)) => (append (cons ?x2 ?y) ?z)
(len nil) => zero
zero => (len nil)
(len (cons ?x3 ?y)) => (succ (len ?y))
(rev nil) => nil
nil => (rev nil)
(rev (cons ?x4 ?y)) => (append (rev ?y) (cons ?x4 nil))
(append (rev ?y) (cons ?x4 nil)) => (rev (cons ?x4 ?y))
(qreva nil ?x) => ?x
(qreva (cons ?z5 ?x) ?y) => (qreva ?x (cons ?z5 ?y))
(qreva ?x (cons ?z5 ?y)) => (qreva (cons ?z5 ?x) ?y)
19:28:16 [0m[33m[WARN] [0m ex is: zero
19:28:16 [0m[33m[WARN] [0m ex is: (succ zero)
19:28:16 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:28:16 [0m[33m[WARN] [0m ex is: nil
19:28:16 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_0 nil)
19:28:16 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:28:16 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:28:16 [0m[33m[WARN] [0mProof mode: true
19:28:16 [0m[33m[WARN] [0m ex is: zero
19:28:16 [0m[33m[WARN] [0m ex is: (succ zero)
19:28:16 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:28:16 [0m[33m[WARN] [0m ex is: nil
19:28:16 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_0 nil)
19:28:16 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:28:16 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:28:16 [0m[33m[WARN] [0mProof mode: true
19:28:16 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 zero) => ?ts_ph_Nat_0
19:28:16 [0m[33m[WARN] [0mproved: (leq zero ?ts_ph_Nat_0) => true
19:28:16 [0m[33m[WARN] [0mproved: (append ?ts_ph_Lst_0 nil) => ?ts_ph_Lst_0
19:28:39 [0m[33m[WARN] [0mproved: (plus ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1)) => (succ (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1))
19:28:39 [0m[33m[WARN] [0mproved: (succ (plus ?ts_ph_Nat_0 ?ts_ph_Nat_1)) => (plus ?ts_ph_Nat_0 (succ ?ts_ph_Nat_1))
19:28:39 [0m[33m[WARN] [0mFound all lemmas
done in 23082
